server/odc-service/src/main/java/com/oceanbase/odc/service/connection/database/DatabaseService.java

```diff
@@ -631,4 +681,23 @@ private String getLockKey(@NonNull Long connectionId) {
         return "DataSource_" + connectionId;
     }
 
+    private Pair<ConnectionSession, Boolean> getConnectionSessionLockUserRequiredPair(
```

Comment:
too complex, avoid return Pair if not neccessary, how about return boolean instead ?

Reply:
okey,  ConnectionSession can be close  after predicate lock user or not